<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fPlayerWidget</class>
 <widget class="QWidget" name="fPlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>110</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>110</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>110</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#fPlayerWidget {
background-color: rgba(0, 0, 0, 0);
background-image: url(:/player_widget/border.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="fPlayerImage">
     <property name="styleSheet">
      <string notr="true">#fPlayerImage {
background-image: url(:/player_avatar/0);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lVotes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>66</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#lVotes {
color: rgb(255, 157, 58);
}</string>
          </property>
          <property name="text">
           <string>99</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lOnline">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>44</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>44</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lRole1">
            <property name="minimumSize">
             <size>
              <width>44</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>44</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lRole2">
            <property name="minimumSize">
             <size>
              <width>44</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>44</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lRole3">
            <property name="minimumSize">
             <size>
              <width>44</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>44</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lPlayerName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 223, 223);</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
